### PR TITLE
cpu/stm32f0: add support for stm32f030cc

### PR DESCRIPTION
**Pull request by niziak**

### CODE REVIEW

File: cpu/stm32f0/include/cpu_conf.h [hunk 1]
```diff
@@ -74,14 +78,15 @@ extern "C" {
  *
  * @{
  */
-#if defined(CPU_MODEL_STM32F091RC) || defined(CPU_MODEL_STM32F072RB)
+
```
Comment (vincent-d) [inline cpu/stm32f0/include/cpu_conf.h]: Extra line

File: cpu/stm32f0/vectors.c [hunk 2]
```diff
@@ -80,7 +81,27 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
...
+    [17] = isr_tim6,                 /* [17] TIM6 global Interrupt */
+    [20] = isr_tim15,                /* [20] TIM15 global Interrupt */
+    [23] = isr_i2c1,                 /* [23] I2C1 Event Interrupt */
+    [24] = isr_i2c2,                 /* [24] I2C2 Event Interrupt */
+    [26] = isr_spi2,                 /* [26] SPI2 global Interrupt */
+    [27] = isr_usart1,               /* [27] USART1 global Interrupt */
+    [28] = isr_usart2,               /* [28] USART2 global Interrupt */
+    [29] = isr_usart3_6,             /* [29] USART3..6 global Interrupt */
+    [31] = isr_usb,                  /* [31] USB global Interrupts & EXTI Line18 Interrupt */
+
```
Comment (vincent-d) [inline cpu/stm32f0/vectors.c]: Extra line

File: cpu/stm32f0/include/cpu_conf.h [hunk 3]
```diff
@@ -61,9 +64,10 @@ extern "C" {
 #define CPU_IRQ_NUMOF                   (28U)
 #elif defined(CPU_MODEL_STM32F051R8) || defined(CPU_MODEL_STM32F091RC)
 #define CPU_IRQ_NUMOF                   (31U)
-#else /* CPU_MODEL_STM32F042K6, CPU_MODEL_STM32F070RB, CPU_MODEL_STM32F072RB */
+#else /* CPU_MODEL_STM32F042K6, CPU_MODEL_STM32F070RB, CPU_MODEL_STM32F072RB, CPU_MODEL_STM32F030CC */
```
Comment (vincent-d) [inline cpu/stm32f0/include/cpu_conf.h]: Please cut line around 80 chars

File: cpu/stm32f0/include/cpu_conf.h [hunk 4]
```diff
@@ -74,14 +78,15 @@ extern "C" {
  *
  * @{
  */
-#if defined(CPU_MODEL_STM32F091RC) || defined(CPU_MODEL_STM32F072RB)
+
+#if defined(CPU_MODEL_STM32F091RC) || defined(CPU_MODEL_STM32F072RB) || defined(CPU_MODEL_STM32F030CC)
```
Comment (vincent-d) [inline cpu/stm32f0/include/cpu_conf.h]: Please cut line around 80 chars

File: cpu/stm32f0/ldscripts/stm32f030cc.ld [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (vincent-d) [inline cpu/stm32f0/ldscripts/stm32f030cc.ld]: I think this file is not needed anymore (ROM and RAM size are computed directly from cpu name)

**Comment by tcschmidt** [comment]
@niziak ready to address the change request of @vincent-d ?

**Comment by PeterKietzmann** [comment]
@niziak please give a quick info after you rebased/addressed comments the next time. Unfortunately the PR needs rebase again. @vincent-d can you check if your comments have been addressed?

**Comment by PeterKietzmann** [comment]
No response for more than half a year. Let's re-open when needed.

**Pull request closed without merging**